mephisto/abstractions/providers/mock/mock_agent.py

```diff
@@ -73,20 +73,45 @@ def get_live_update(self, timeout=None) -> Optional[Dict[str, Any]]:
             self.datastore.agent_data[self.db_id]["acts"].append(act)
         return act
 
-    def approve_work(self) -> None:
+    def approve_work(
+        self,
+        feedback: Optional[str] = None,
+        bonus: Optional[str] = None,
+        skip_unit_review: bool = False,
+    ) -> None:
         """
         Approve the work done on this specific Unit
 
         Mock Units
         """
         self.update_status(AgentState.STATUS_APPROVED)
 
-    def reject_work(self, reason) -> None:
+        if not skip_unit_review:
+            unit = self.get_unit()
+            self.db.new_unit_review(
+                unit_id=int(unit.db_id),
+                task_id=int(unit.task_id),
+                worker_id=int(unit.worker_id),
```

Comment:
The cast to int should occur on the db side, as the DB is what is determining that these are ints vs being random string IDs.